config/defaults/config.LINUX_INTEL.mk

```diff
@@ -1,5 +1,19 @@
-FCOMPILER=mpiifort
+# ----------------------------------------------------------------------
+# Config file for Intel ifort
+# ----------------------------------------------------------------------
+
+# ------- Define a possible parallel make ------------------------------
+PMAKE = make -j 4
```

Comment:
This will not do anything unless `PMAKE` is used in the Makefiles, which does not look like the case